os/linux

```diff
@@ -119,9 +119,7 @@ case $OS in
         sudo dnf groupinstall -y "C Development Tools and Libraries"
         dnf_add_mongo_repo
         sudo dnf -y install $FEDORA_PACKAGES
-        dnf_add_and_install_docker_ce
         fedora_enable_databases_on_restart
-        systemd_enable_docker_on_restart
```

Comment:
~~Is there a reason docker was removed?~~ Never mind read the PR description. Does podman work with the boilerplate projects docker files? I'm curious.

Reply:
I think it should, though I haven't tested it yet though. `podman` is supposed to be complete compatible with docker commands, most people `alias docker=podman` on fedora. There is also a `podman-compose`

Besides docker doesn't maintain repositories for the latest fedora anyway, so that part would always break.